tutorials/export/exporting_basics.rst

```diff
@@ -338,11 +347,11 @@ your system and the location of the keystore you just created.
 
 .. image:: img/export_editor_android_settings.png
 
-Now you're ready to export. Click on *Project -> Export* and add a preset
-for Android (see above). Select the Android Presets and under *Options* go to
-*Screen* and set *Orientation* to "Portrait".
+Now you're ready to export. Click on **Project -> Export** and add a preset
+for Android (see above). Select the Android Presets and under **Options** go to
+**Screen** and set **Orientation** to **Portrait**.
 
```

Comment:
"This step is optional and will depend on your game" Our game demo is design to work in Portrait mode, so we use this setting.